test/controllers/dashboard/internship_offers/destroy_test.rb

```diff
@@ -29,6 +38,19 @@ class DestroyTest < ActionDispatch::IntegrationTest
       assert_equal 'Votre annonce a bien été supprimée', flash[:success]
     end
 
+    test 'DELETE #destroy as statistician owning internship_offer updates internship_offer' do
+      # internship_offer = create(:weekly_internship_offer_by_statistician)
```

Comment:
can't call this Trait and don't know why.
Any idea @mfo ?

Reply:
oui, l'usage d'un trait va ac une factory. ex: 
create(:weekly_internship_offer, :by_statistician)

Reply:
@mfo weekly_internship_offer L24 est un aussi un trait mais il est appelé sans pb